docs/src/main/paradox/akka-management.md

```diff
@@ -147,7 +120,9 @@ base path. For example, when using Akka Cluster Management routes the members in
 
 @@@ note
 
-HTTPS is not enabled by default as additional configuration from the developer is required This module does not provide security by default. It's the developer's choice to add security to this API.
+HTTPS is not enabled by default as additional configuration from the developer is required This module does not provide security by default.
+It's the developer's choice to add security to this API. It is generally advisable not to expose management endpoints
+on the internet.
```

Comment:
Sounds very arpanet, might we say something indicating publicly or not on a private network vs this word?